client/src/Components/Settings/actionCreator.js

```diff
@@ -9,19 +9,37 @@ import config from "../../config/index";
 // keep Store updated with user info. If it changes, update
 // use the billing plan ID to query billing table and send that to Stripe
 
+export const getUser = user => {
+  return async dispatch => {
+    dispatch({ type: actions.USER_STARTED });
+    try{
+      const getUser = await axios.get(`${config.apiUrl}/api/users/me`)
```

Comment:
`getUser` might be confusing because the getUser variable is already declared in the upper scope and is the name of the function. Maybe you could just call it `user`?